papis/downloaders/thesesfr.py

```diff
@@ -12,66 +12,23 @@ def __init__(self, url: str) -> None:
 
     @classmethod
     def match(cls, url: str) -> Optional[papis.downloaders.Downloader]:
-        if re.match(r".*theses.fr.*", url):
+        if re.match(r".*theses.fr.*|\d{4}[a-zA-Z]{3,}\d+", url):
```

Comment:
Is this a documented format for their identifiers? Maybe add a little comment with a link to something?